packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -5,14 +5,17 @@
 //  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 //  */
 import * as events from 'events';
-import * as os from 'os';
+import * as path from 'path';
+import { mkdir } from 'shelljs';
 import * as vscode from 'vscode';
+import { nls } from '../messages';
 import { ReadableApexTestRunExecutor } from './readableApexTestRunExecutor';
 import {
   ApexTestNode,
   ApexTestOutlineProvider,
   TestNode
 } from './testOutlineProvider';
+import pathExists = require('path-exists');
```

Comment:
Change this to use `from` instead of `require` and (if possible) just load the function/method you are going to use e.g. sync.

Reply:
Okay will do!